Options.py

```diff
@@ -1409,6 +1415,100 @@ def verify(self, world: typing.Type[World], player_name: str, plando_options: "P
             link["item_pool"] = list(pool)
 
 
+class PlandoItem(typing.NamedTuple):
+    items: typing.Union[typing.List[str], typing.Dict[str, typing.Any]]
+    locations: typing.List[str]
+    world: typing.Union[int, str, bool, None, typing.Iterable[str], typing.Set[int]] = False
+    from_pool: bool = True
+    force: typing.Union[bool, typing.Literal["silent"]] = "silent"
+    count: typing.Union[int, bool, typing.Dict[str, int]] = False
+    percentage: int = 100
+
+
+class PlandoItems(Option[typing.List[PlandoItem]]):
+    """Generic items plando."""
+    default = ()
+    supports_weighting = False
+    display_name = "Plando Items"
+
+    def __init__(self, value: typing.Iterable[PlandoItem]) -> None:
+        self.value = list(deepcopy(value))
+        super().__init__()
+
+    @classmethod
+    def from_any(cls, data: typing.Any) -> Option[typing.List[PlandoItem]]:
+        if not isinstance(data, typing.Iterable):
+            raise Exception(f"Cannot create plando items from non-Iterable type, got {type(data)}")
+
+        value: typing.List[PlandoItem] = []
+        for item in data:
+            if isinstance(item, typing.Mapping):
+                percentage = item.get("percentage", 100)
+                if not isinstance(percentage, int):
+                    raise Exception(f"Plando `percentage` has to be int, not {type(percentage)}.")
+                if roll_percentage(percentage):
+                    count = item.get("count", False)
+                    items = item.get("items", [])
+                    if not items:
+                        items = item.get("item", None)  # explicitly throw an error here if not present
+                        if not items:
+                            raise Exception("You must specify at least one item to place items with plando.")
+                        items = [items]
```

Comment:
This removed the ability for `item` to be a dictionary. Could maybe do this, but it's kind of ugly
```suggestion
                        if isinstance(items, str):
                            items = [items]
                        elif isinstance(items, dict):
                            count = 1
                        else:
                            raise Exception(f"Plando 'item' has to be string or dictionary, not {type(items)}.")
```